plugins/Python4Capella/sample_scripts/xAB Functional Content.py

```diff
@@ -0,0 +1,214 @@
+# name                 : Extract content of xAB diagram
+# script-type          : Python
+# description          : Extract content of xAB diagram
+# popup                : enableFor(org.eclipse.sirius.viewpoint.DRepresentationDescriptor)
+
+'''
+This script allows to exxtract content of xAB diagram
+It will create a folder result in the selected Capella project with the resulting xlsx file.
+'''
+
+# To run it:
```

Comment:
To run it you need to right click on a DRepresentationDescriptor (diagram in the project explorer ?) and select the "Extract content of xAB diagram" menu.